### DISCUSSION TITLE
HappyModel EP1 Receiver SBUS and CRSF Telemetry in

### DISCUSSION THREAD

**Discussion by gfalgiano** [ExpressLRS/ExpressLRS#3051] [Q&A] [answered]
I have a small plane where I send out SBUS from the EP1 receiver to a simple stabilizer, which then outputs PWM to the servos. I'd like to use the remaining pin on the EP1 to input a CRSF telemetry signal and send the data back to my radio.

I see that with the PWM receivers, there is the serial2 option, but I don't see this for the EP1. Any suggestions on how I could make this work? I'm hoping to add a sensor from this project: https://github.com/dgatf/msrc

**Comment by JyeSmith** [comment, marked as answer]
It unfortunately wont be possible to split the uart for sbus out and crsf in.

Your best chance is to use only crsf on the EP and a crsf to sbus converter to the gyro.
